site/index.css

```diff
@@ -60,10 +62,18 @@ body {
 
 @media screen and (max-width: 620px) {
   #aquapants-header h1 {
-    font-size: 2.5em;
+    font-size: 3.2em;
+  }
+
+  #aquapants-header p {
+    font-size: 1.2em;
+  }
+
+  #aquapants-quote {
```

Comment:
I think this is a good implementation, but I don't like how it looks on taller screens (too crowded). Can we use a media query to only apply this rule to mobile horizontal screens (ie. max-height is 620px)?

Reply:
Yes I can do that, I thought how much I can put in that max-width:620px query 😅 cause that that what the requirement is for (landscape mode)

Reply:
Ah I see, maybe I should have commented that block. When I added the `max-width:620px` query, I meant for that to be portrait mode. The idea of using 620px as a breakpoint was loosely based on [this article](https://www.freecodecamp.org/news/the-100-correct-way-to-do-css-breakpoints-88d6a5ba1862/) and some arguments I read on StackOverflow regarding common breakpoints.

I think since landscape mode is just rotated portrait-mode, we should simply treat it with a `max-height:620px` media query. Your current implementation does not apply the style on most landscape-orientation phones (which often have a width (ie. the height of the phone in portrait mode) that is larger than 620px.